packages/renderer/src/App.svelte

```diff
@@ -58,6 +60,15 @@ onMount(() => {
   });
 });
 
+onMount(async () => {
+  const welcomeUtils = new WelcomeUtils();
```

Comment:
I would put this inside the welcome screen page

So the component manages it's display not his parent